data/src/main/java/com/linkedin/data/template/RecordTemplate.java

```diff
@@ -499,7 +515,7 @@ private boolean checkPutNullValue(RecordDataSchema.Field field, Object object, S
     return _cache;
   }
 
-  private DataMap _map;
+  protected DataMap _map;
```

Comment:
Is it possible to just reduce this to being package-private? Don't think we'd want this field to be settable by subclasses.

Reply:
We can't. Generated subclasses need this.